server/utils/verifiers/EdanVerifier.ts

```diff
@@ -0,0 +1,426 @@
+                            } else {
+                                LOG.info(`${this.config.logPrefix} successfully updated Packrat unit to match EDAN (id:${systemObject.idSystemObject} | subject:${subject.Name})`, LOG.LS.eSYS);
+                                str += 'EDAN subject. updated Packrat unit to match EDAN';
+                            }
+                        } else {
+                            str += 'EDAN subject. automatic updating disabled. rerun or manual fix required';
+                        }
+                    }
+
+                    // store for output
+                    output.push(str);
+                } else {
+                    LOG.info(`${this.config.logPrefix} Unit compare succeeded! (id:${systemObject.idSystemObject} | subject:${subject.Name})`, LOG.LS.eSYS);
+                    let str = outputPrefix;
+                    str += 'success,';
+                    str += 'units,';
+                    str += ',';
+                    str += '"' + packratUnit.Abbreviation + ' - ' + packratUnit.Name + '",';
+                    str += '"' + edanUnit.Abbreviation + ' - ' + edanUnit.Name + '",';
+                    output.push(str);
+                }
+            } else if(packratUnit && !edanUnit) {
+                LOG.error(`${this.config.logPrefix} Packrat unit not found in EDAN record (id:${systemObject.idSystemObject} | subject:${subject.Name} | packrat:${JSON.stringify(packratUnit)})`, LOG.LS.eSYS);
+                let str = outputPrefix;
+                str += 'error,';
+                str += 'units,';
+                str += 'Packrat unit not found in EDAN,';
+                str += '"' + packratUnit.Abbreviation + ' - ' + packratUnit.Name + '",';
+                str += 'null,';
+                str += ((isDPOSubject)?'DPO created subject. needs manual fix to apply to EDAN.':'EDAN subject. needs manual fix because EDAN should have unit assigned. (todo)')+',';
+                output.push(str);
+            } else if(!packratUnit && edanUnit) {
+                LOG.error(`${this.config.logPrefix} EDAN unit not found in Packrat (id:${systemObject.idSystemObject} | subject:${subject.Name} | EDAN:${edanUnit.Name})`, LOG.LS.eSYS);
+                let str = outputPrefix;
+                str += 'error,';
+                str += 'units,';
+                str += 'EDAN unit not found in Packrat,';
+                str += 'null,';
+                str += '"' + edanUnit.Abbreviation + ' - ' + edanUnit.Name +'",';
+
+                // apply edan unit to packrat unit (creating new unit in process)
+                if(isDPOSubject) {
+                    str += 'DPO subject. needs manual fix as Packrat should have unit';
+                } else {
+                    if(this.config.fixErrors) {
+                        const replaceUnitResult: boolean = await this.replacePackratUnit(packratUnit,edanUnit);

```

Comment:
Now that I see how `replacePackratUnit` is used, instead of updating our Unit record, we want to update the Subject record, and point it at the correct idUnit for the Edan Unit.